pkg/controllers/rollout/controller.go

```diff
@@ -157,16 +171,34 @@ func (r *Reconciler) Reconcile(ctx context.Context, req runtime.Request) (runtim
 		// Here it will correct the binding status just in case this happens last time.
 		return runtime.Result{}, r.checkAndUpdateStaleBindingsStatus(ctx, allBindings)
 	}
-	klog.V(2).InfoS("Picked the bindings to be updated", "clusterResourcePlacement", crpName, "numberOfBindings", len(toBeUpdatedBindings), "numberOfStaleBindings", len(staleBoundBindings))
-
-	// StaleBindings is the list that contains bindings that need to be updated but are blocked by the rollout strategy.
-	// Update the status first, so that if the rolling out (updateBindings func) fails in the middle, the controller will
-	// recompute the list and the result may be different.
+	klog.V(2).InfoS("Picked the bindings to be updated",
+		"clusterResourcePlacement", crpName,
+		"numberOfToBeUpdatedBindings", len(toBeUpdatedBindings),
+		"numberOfStaleBindings", len(staleBoundBindings),
+		"numberOfUpToDateBindings", len(upToDateBoundBindings))
+
+	// StaleBindings is the list that contains bindings that need to be updated (binding to a
+	// cluster, upgrading to a newer resource/override snapshot, or removing) but are blocked by
+	// the rollout strategy.
+	//
+	// Update the status first, so that if the rolling out (updateBindings func) fails in the
```

Comment:
what does this comment mean?

Reply:
Hi Ryan! This (`Update the status first...`) was the original comment I believe.